File: app/src/main/java/com/sinata/framework/arithmatic/ArithmaticActivity.java
package com.sinata.framework.arithmatic;

import android.os.Bundle;
import android.widget.TextView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

/**
 * @author cjq
 * @Date 11/5/2021
 * @Time 11:34 AM
 * @Describe:
 */
public class ArithmaticActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(new TextView(this));
//        Solution.removeElement(new int[]{1,2,2,2,4},2);
    }
}

File: app/src/main/java/com/sinata/framework/arithmatic/ArithmaticActivity.kt
package com.sinata.framework.arithmatic

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import com.sinata.framework.R

class ArithmaticActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(TextView(this))
        Solution.isValid("()[]{}")
    }
}